project/build/dev/d8/themes/custom/patternlab/source/_meta/_00-head.twig

```diff
@@ -15,6 +15,7 @@
   <link rel="stylesheet" href="../../../../dest/style.css">
   <!-- endinject -->
   <link rel="stylesheet" href="../../../../source/styleguide/custom-styleguide-specific.css">
-
+  <script src="https://use.typekit.net/jhz5ozy.js"></script>
```

Comment:
It's two faces  Modesto-Condensed Bold for the Post Headers & Source-Sans-Pro regular, bold, and light for body copy, strong, light, for Section Headings & Meta info